index.html

```diff
@@ -14,8 +14,16 @@
         <header>
             <div class="toggle-menu">
                 <h3 class="welcome"><a href="">Welcome</a></h3>
-                <nav class="small">
-                    <i class="fa-solid fa-bars"></i>
+               
+                <!--mobile-menu display only on mobile-->
+                <i class="fa-solid fa-bars small" id="hambuger"></i>
+                <nav  class="mobile-menu" id="mobileMenu">
+                    <i class="fa-solid fa-x" id="x"></i>
```

Comment:
- [ ] On your desktop view, your hamburger button displays on top of your beautiful design 😉 . Adjust your javascript and CSS styling to make sure it's not visible on the desktop. Imagine visiting a normal website and seeing this kind of feature, you'll instantly know it's a bug 😄 . Now be that developer that fixes such just as the world-class dev you're becoming 👏 


# View on Desktop:
![image](https://user-images.githubusercontent.com/50990925/215664631-5d66ee22-91e2-438f-86b7-94768b7f419f.png)